_gsocproposals/2019/proposal_ActsGpuRace.md

```diff
@@ -0,0 +1,72 @@
+---
+title: GPU Race for Particle Hunting
+layout: gsoc_proposal
+project: ACTS
+year: 2019
+organization: LAL
+---
+
+## Description
+
+[ACTS](http://acts.web.cern.ch) is a free and open-source software project for
```

Comment:
I see you have pushed some commits to change from `ACTS` to `Acts`.  There are some places on this documents where the Acts projects appears in upper case. Should those be also changed for consistency?